.storybook/InputField.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import styled from 'styled-components'
+import { Box, theme, Flex, Text, InputField, Input } from '../src'
```

Comment:
You are not using `theme` anywhere here.

Reply:
I think that was copy pasta